buildSrc/build.gradle

```diff
@@ -46,7 +46,7 @@ task sourcesJar(type: Jar) {
     classifier = 'sources'
 }
 
-def publishVersion = '0.20.7'
+def publishVersion = '0.20.8-SNAPSHOT'
```

Comment:
you can do so if you want to locally. Even if we wanted to start doing that, it should be a separate change